modules/widgets/social-icons/social-icons-admin.css

```diff
@@ -0,0 +1,96 @@
+.jetpack-social-icons-widget-item {
+	background: #fff;
+	border: 1px solid #e5e5e5;
+	cursor: move;
+	margin: 0;
+}
+
+html[class*='wordpress_com'] .jetpack-social-icons-widget-item,
+.in-calypso .jetpack-social-icons-widget-item {
+	border-color: #c8d7e1;
+}
+
+.jetpack-social-icons-widget-item:hover {
+	outline: 1px solid #999;
+	outline-offset: -1px;
+}
+
+html[class*='wordpress_com'] .jetpack-social-icons-widget-item:hover,
+.in-calypso .jetpack-social-icons-widget-item:hover {
+	outline-color: #a8bece;
+}
+
+.jetpack-social-icons-widget-item.ui-sortable-helper {
+	border-color: #999;
+}
+
+html[class*='wordpress_com'] .jetpack-social-icons-widget-item.ui-sortable-helper,
+.in-calypso .jetpack-social-icons-widget-item.ui-sortable-helper {
+	border-color: #a8bece;
+}
+
+.jetpack-social-icons-widget-item.ui-sortable-helper {
+	-webkit-box-shadow: 0 1px 2px rgba(0, 0, 0 , 0.1);
```

Comment:
The most recent browser that required the `-webkit-` prefix here was the Android 3 browser from 2011.  I don't think we need to include this unless there's a strange unexpected edge case? https://caniuse.com/#search=box-shadow